admin/server/middleware/browserify.js

```diff
@@ -67,17 +77,24 @@ module.exports = function (file, name) {
 		if (devWriteDisc) {
 			opts.fullPaths = true;
 		}
-		if (name) {
+		if (/packages\.js/.test(outputPath)) {
```

Comment:
this is one of the complexities I'd like to refactor out. but maybe packages is complex enough that it can just live by itself rather than reusing this file?